### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by amaanvaddadi**

### CODE REVIEW

File: client/components/Home.js [hunk 1]
```diff
@@ -1,57 +1,60 @@
 import React, { Component } from 'react'
 import { connect } from 'react-redux'
 import { bindActionCreators } from 'redux'
-import { fetchRecipe } from '../actions/actionCreators'
+import { fetchRecipe} from '../actions/actionCreators'
 import { Router, Route, hashHistory, Link } from 'react-router'
 import RecipeList from './RecipeList'
+import ReactPlayer from 'react-player'
```
Comment (mandy-kate) [inline client/components/Home.js]: remove this please

File: client/components/Home.js [hunk 2]
```diff
@@ -1,57 +1,60 @@
...
 class Home extends Component {
 
+  constructor(props){
 
+    super(props)
+    this.handleClickImage= this.handleClickImage.bind(this)
+  }
+
+  handleClickImage(){
+    console.log('bananas');
```
Comment (mandy-kate) [inline client/components/Home.js]: remove bananas

File: client/components/Recipe.js [hunk 3]
```diff
@@ -96,89 +96,78 @@ class Recipe extends React.Component {
...
-  } else {
-    console.log(this.props.data.listening)
-    return (
-      <div> <img width="100" height="100" src='../images/listening.png' alt='listening_green' />
-      <h5>I am now listening</h5>
-      <h5>Say Next or click the next button</h5>
-    </div>
-  )}
+    }else {
+      console.log(this.props.data.listening)
```
Comment (mandy-kate) [inline client/components/Recipe.js]: pls remove logs

File: client/components/Recipe.js [hunk 4]
```diff
@@ -96,89 +96,78 @@ class Recipe extends React.Component {
...
+      return <div className="thumbnail spinner spinner-4"> <img  width="300" height="400" src={recipePage_image_path} alt='listening_green' onClick={this.startListening}/>
+      {this.isListening}
```
Comment (mandy-kate) [inline client/components/Recipe.js]: refactor with variable for spinner and color and alt text and only return once

File: database/migrations/20160908100945_createRecipiesTable.js [hunk 5]
```diff
@@ -9,7 +9,8 @@ exports.up = function(knex, Promise) {
     table.integer('length');
     table.text('whole_recipe_audio_path');
     table.text('ingredients_audio_path');
-    table.text('image_path');
+    table.text('recipeList_image_path');
+    table.text('recipePage_image_path')
```
Comment (mandy-kate) [inline database/migrations/20160908100945_createRecipiesTable.js]: this needs to be in a different migration

File: client/components/Home.js (hunk 1 above)
Comment (amaanvaddadi) [inline client/components/Home.js]: Done!

File: client/components/Home.js (hunk 2 above)
Comment (amaanvaddadi) [inline client/components/Home.js]: done!

File: client/components/Recipe.js (hunk 3 above)
Comment (amaanvaddadi) [inline client/components/Recipe.js]: done! also refactored listening

**Pull request merged**